alibi/utils/frameworks.py

```diff
@@ -36,12 +37,12 @@ def tensorflow_installed() -> bool:
     if not has_tensorflow:
         raise ImportError(template.format(pkg=tf_required))
     if int(tf_version[0]) > 1:
```

Comment:
We need to introduce here the change we discussed privately, otherwise the docs do not build.